tests/basic.rs

```diff
@@ -873,14 +875,14 @@ fn message_channel_debug() {
     assert_eq!(
         format!("{:?}", mc),
         "MessageChannel<basic::Hello, alloc::string::String>(\
-            Sender<basic::Greeter> { rx_count: 1, tx_count: 1, rc: TxStrong(()) }\
+            ChanPtr<basic::Greeter, xtra::inbox::chan_ptr::TxStrong> { rx_count: 1, tx_count: 1 }\
```

Comment:
I wonder if we should be leaking this impl detail. This is just a general comment, though, as we were leaking Sender before anyway :smile:

Reply:
No preference here really. Can we open an issue, add it to the milestone and defer that decision? :)

Reply:
Sure!